python/src/pyarrow/parquet.cc

```diff
@@ -0,0 +1,30 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
what are the purpose of this file and the next, it seems that they have overlapping imports and no code?